services/federator/src/Federator/Options.hs

```diff
@@ -62,9 +62,21 @@ data RunSettings = RunSettings
   { -- | Would you like to federate with everyone or only with a select set of other wire-server installations?
     federationStrategy :: FederationStrategy,
     useSystemCAStore :: Bool,
-    remoteCAStore :: Maybe FilePath
+    remoteCAStore :: Maybe FilePath,
+    clientCertificate :: Maybe FilePath,
+    clientPrivateKey :: Maybe FilePath
   }
-  deriving (Show, Generic)
+  deriving (Eq, Show, Generic)
+
+defRunSettings :: RunSettings
+defRunSettings =
+  RunSettings
+    { federationStrategy = AllowAll,
```

Comment:
I think the default should be no federation (allow list with empty list) at least at this stage, no?

Tests is another matter, there could be a default test run settings object under ./test/ somewhere.

Reply:
Yes, this should only be used for tests. I moved it to a test module (8878a177c94020ce1270ddeb4e2995c1bfbdb253).